docs/source/activelearning.rst

```diff
@@ -7,12 +7,25 @@ Active learning denotes the scenario in which a reviewer actively provides class
+Query Strategies
+~~~~~~~~~~~~~~~~
+
+The model can adopt different strategies in deciding which publication should be presented next.
+Certainty-based active learning, also known as max sampling, is typically the most suited to the scenario in which we are dealing with highly imbalanced data sets and finding all relevant references as soon as possible is quintessential to success [4]. This is because with certainty-based active learning, the model presents those references to the reviewer which are most likely to be inclusions first, thereby expediting the process of finding all of the relevant references. 
 
 Traditionally, uncertainty sampling or uncertainty-based active learning is used in case training the best model is desired. Using uncertainty sampling, those references on which the model's classifications are the least certain will be presented to the reviewer first. However, in the case where time-saving is of the essence, uncertainty-based active learning may not be the optimal strategy [4]. Although uncertainty-based active learning generally leads to a more robust model in the end because it learns the most from these uncertain classifications, this gain is not significant [4]. This entails that uncertainty-based active learning may not be suited to a scenario in which we want to find the relevant references as soon as possible, such as with abstract screening for systematic reviews [4].
 
 Ultimately, the active learning cycle has to stop at some point and finding a suitable stopping criterion for this active learning cycle has proven to be challenging. However, new methods to decide the ideal moment to stop are currently being developed [2].
 
+Word Embeddings
+~~~~~~~~~~~~~~~
+To predict the class of a publication class (e.g. whether a publication should be included or excluded), the model uses information from the publications in the dataset. Examples of such information are titles and abstracts. However, the model cannot predict the publication class from titles and abstracts as they are. Their textual content to needs to be mapped to feature vectors. This process of numerically representing textual content is called 'word embeddings'. 
+
+A classical example of word embeddings a 'bag of words' (bow) representation. For each text in the data set, the number of occurrences of each word is stored. This leads to n features, where n is the number of distinct words in the texts. The bag-of-words method is simplistic and will highly value often occuring but otherwise meaningless words such as "and". A more sophisticated approach is Term-frequency Inverse Document Frequency (TF-IDF). TF-IDF circumvents this problem by adjusting the term frequency in a text with the inverse docuement frequency, the frequency of a given word in the entire data set [7].
```

Comment:
Document vector representation (or Vector representations for abstracts and titles)

To predict whether a publication should be included or excluded in a systematic review, one important step is to convert the textual information in the data set to a representation. Vector representation models have been used to convert a document (sequence of words) to vectors of numbers [REF]. These models include two main techniques: bag of words and distributed representation for words.
Word embeddings or neural embeddings has been proposed to represent words as dense vectors that are derived by various training methods inspired from neural-network language modeling [REF].
…
The BOW models are vectorization techniques that disregard grammar and word orders, but use the occurrences of words within a document [REF]. These models are essentially categorized in two approaches: binary and frequency-based methods. Term-frequency (TF) and Term-frequency Inverse Document Frequency (TF-IDF) are two examples of the frequency-based BOW methods.
…

Reply:
Nice! What exactly do you suggest? Replace this part with the text you suggest? 

Do you know which publications you want to refer to?

Reply:
I would replace with the new text and add more details. Examples for refs:

Mikolov, T., Sutskever, I., Chen, K., Corrado, G.S. and Dean, J., 2013. Distributed representations of words and phrases and their compositionality. In Advances in neural information processing systems (pp. 3111-3119).

Le, Q. and Mikolov, T., 2014, January. Distributed representations of sentences and documents. In International conference on machine learning (pp. 1188-1196).

Turney, P.D. and Pantel, P., 2010. From frequency to meaning: Vector space models of semantics. Journal of artificial intelligence research, 37, pp.141-188.
 

Reply:
Should we perhaps also include some extra information on e.g. skip-gram and how it allows for training word embeddings? If so, I can write that if you want

Reply:
Also, do we want to include the piece on word embeddings in the active learning section, or should we move this to a new file dedicated to word embeddings?